contracts/src/v0.8/dev/vrf/VRFConsumerBaseV2Plus.sol

```diff
@@ -103,7 +103,7 @@ abstract contract VRFConsumerBaseV2Plus is IVRFMigratableConsumerV2Plus, Confirm
   error OnlyOwnerOrCoordinator(address have, address owner, address coordinator);
   error ZeroAddress();
 
-  IVRFMigratableCoordinatorV2Plus internal s_vrfCoordinator;
+  IVRFMigratableCoordinatorV2Plus public s_vrfCoordinator;
```

Comment:
Does this really need to be public if subclasses are using it? It should be accessible to subclasses without changing visibility to public.

Reply:
this is needed in order for other coordinator to be updated during the migration process. 

this was discussed with @jinhoonbang - [slack](https://chainlink-core.slack.com/archives/C03FPM5RL9J/p1694052396948479?thread_ts=1694043917.195299&cid=C03FPM5RL9J)

Reply:
I think it's fine to leave it `internal` and migration will still work. 

Only reason for changing it to public was to allow querying for coordinator address. Alternatively, we could add a `getVRFCoordinator()` in base consumer or concrete consumer but I think this is fine as is

Reply:
Ah got it, so this is really for external parties to query the VRF coordinator set on a consumer then. Makes sense.